src/compat/posix/pwd_grp/pwd_grp.c

```diff
@@ -297,35 +325,46 @@ int getgrnam_r(const char *name, struct group *grp,
 	}
 
 	fclose(file);
-
-	if (0 != res) {
-		result = NULL;
-		return res == ENOENT ? 0 : -1;
-	}
-	return 0;
-
+	return res;
```

Comment:
http://pubs.opengroup.org/onlinepubs/9699919799/functions/getgrnam.html

`The getgrnam_r() function shall return zero on success or if the requested entry was not found`

So we shall return 0 if the requested entry was not found.

It also breaks ssh.

Reply:
Done